src/main/java/com/specialwarriors/conal/github_repo/controller/GithubRepoController.java

```diff
@@ -72,7 +72,7 @@ public String getRepositoryId(@SessionAttribute Long userId,
 
     @PostMapping("/{repositoryId}")
     public String deleteRepository(@SessionAttribute Long userId,
```

Comment:
```suggestion
    @DeleteMapping("/{repositoryId}")
    public String deleteRepository(@SessionAttribute Long userId,
```
`POST`보다 `DELETE`가 적절해보이는데 검토 부탁드립니다~

Reply:
HTML Form으로는 GET, POST 방식의 요청만 가능하고, PUT,DELETE의 실제 요청은 POST로 전송되기 때문에 단순히 method="DELETE" 같이 작성한다고 PUT, DELETE 메소드 사용이 안된다고 합니다.

이를 위해 form 내부의 같이 hidden 타입의 input을 작성하고, httpMethodFilter를 이용하여 request로부터 정보를 읽어와 PUT, DELETE로 구분할 수 있다는데, httpMethodFilter를 이용해서 코드 다시 짜볼까요?